terraform/global-resources/locals.tf

```diff
@@ -0,0 +1,9 @@
+locals {
+  global_resources {
+    application-name    = "Modernisation Platform - Global Resources"
+    business-unit       = "Ministry of Justice - Hosting"
+    tech-contact-email  = "modernisation-platform@digital.justice.gov.uk"
+    budget-holder-email = "steve.marshall@digital.justice.gov.uk"
+    stage               = "production"
+  }
+}
```

Comment:
Some of these look like things that would be covered by [our tagging guidance](https://ministryofjustice.github.io/technical-guidance/documentation/standards/documenting-infrastructure-owners.html).